packages/application-shell/src/components/login/login.js

```diff
@@ -170,70 +171,78 @@ export class Login extends React.PureComponent {
+                  <FormattedMessage {...messages.title} />
+                </Text.Headline>
                 {this.renderErrorMessage(
                   !formikProps.isSubmitting && this.state.error
                 )}
-                <Text.Subheadline elementType="h4">
-                  <FormattedMessage {...messages.title} />
-                </Text.Subheadline>
-                <TextField
-                  name="email"
-                  title={this.props.intl.formatMessage(messages.email)}
-                  isAutofocussed={true}
-                  isRequired={true}
-                  value={formikProps.values.email}
-                  touched={formikProps.touched.email}
-                  errors={formikProps.errors.email}
-                  onChange={formikProps.handleChange}
-                  onBlur={formikProps.handleBlur}
-                  isDisabled={formikProps.isSubmitting}
-                  renderError={this.renderEmailErrors}
-                />
-                <PasswordField
-                  name="password"
-                  title={this.props.intl.formatMessage(messages.password)}
-                  isRequired={true}
-                  value={formikProps.values.password}
-                  touched={formikProps.touched.password}
-                  errors={formikProps.errors.password}
-                  onChange={formikProps.handleChange}
-                  onBlur={formikProps.handleBlur}
-                  isDisabled={formikProps.isSubmitting}
-                />
-                <Spacings.Inline>
+                <Spacings.Stack scale="m">
+                  <TextField
+                    name="email"
+                    value={formikProps.values.email}
+                    title={this.props.intl.formatMessage(messages.email)}
+                    onBlur={formikProps.handleBlur}
+                    errors={formikProps.errors.email}
+                    touched={formikProps.touched.email}
+                    onChange={formikProps.handleChange}
+                    isRequired={true}
+                    isDisabled={formikProps.isSubmitting}
+                    renderError={this.renderEmailErrors}
+                    placeholder={this.props.intl.formatMessage(messages.email)}
+                    isAutofocussed={true}
+                  />
+                  <Spacings.Stack>
+                    <PasswordField
+                      name="password"
+                      value={formikProps.values.password}
+                      title={this.props.intl.formatMessage(messages.password)}
+                      onBlur={formikProps.handleBlur}
+                      errors={formikProps.errors.password}
+                      touched={formikProps.touched.password}
+                      onChange={formikProps.handleChange}
+                      isRequired={true}
+                      isDisabled={formikProps.isSubmitting}
+                    />
+                    <Countdown
```

Comment:
This can be removed now that we will have a an actual page for reset password

Reply:
But should we remove this in this PR or only when we actually have the page?